static/app/components/arithmeticBuilder/token/function.tsx

```diff
@@ -347,7 +669,8 @@ function InternalInput({
           )
         }
       </ComboBox>
-    </Fragment>
+      {argumentIndex < functionToken.attributes.length - 1 && ','}
```

Comment:
This feels like it should be the responsibility of the wrapper component to render and not the input.